apps/website/src/pages/about/tech/commands.tsx

```diff
@@ -326,62 +330,115 @@ const AboutTechPage: NextPage = () => {
+            <div className="col-span-1 lg:col-span-3">
+              {selectedCamera && (
+                <Fragment key={selectedCamera}>
+                  <Heading
+                    level={3}
+                    className="scroll-mt-14 text-2xl"
+                    id={`presets:${camelToKebab(selectedCamera)}`}
+                  >
+                    {presets[selectedCamera]?.title}
+                    <span className="text-sm text-alveus-green-400 italic">
+                      {` (${selectedCamera.toLowerCase()})`}
+                    </span>
+                  </Heading>
+
+                  <div className="mt-4 grid grid-cols-2 gap-4 sm:grid-cols-3 lg:grid-cols-4">
+                    {typeSafeObjectEntries(
+                      presets[selectedCamera]?.presets ?? {},
+                    ).map(([name, preset]) => {
+                      return (
                         <div
                           key={name}
-                          className="group/preset mb-4 flex flex-col items-baseline lg:mb-0 lg:flex-row lg:gap-2"
+                          className="overflow-hidden rounded-lg border shadow-lg"
                         >
-                          <dt>
-                            <pre>
-                              <code className="text-sm">
-                                <span className="opacity-40 group-first/preset:opacity-100">
-                                  {`!ptzload ${camera.toLowerCase()} `}
-                                </span>
-                                {name}
-                              </code>
-                            </pre>
-                          </dt>
-
-                          <dd className="flex items-center gap-1">
-                            <CopyToClipboardButton
-                              text={`!ptzload ${camera.toLowerCase()} ${name}`}
-                            />
-                            <RunCommandButton
-                              command="ptzload"
-                              args={[camera.toLowerCase(), name]}
-                              subOnly
-                            />
-                            <p className="text-sm text-alveus-green-400 italic">
+                          <div className="group relative aspect-video">
+                            {preset.image ? (
+                              <Image
+                                src={preset.image}
+                                alt={preset.description}
+                                fill
+                                className="object-cover transition-transform group-hover:scale-105"
+                              />
+                            ) : (
+                              <div className="flex h-full w-full items-center justify-center bg-alveus-green-50 text-xs text-alveus-green-300">
+                                No Image
+                              </div>
+                            )}
+                          </div>
```

Comment:
```suggestion
                          {preset.image ? (
                            <Image
                              src={preset.image}
                              alt={preset.description}
                              width={270}
                              className="aspect-video w-full object-cover transition-transform hover:scale-105"
                            />
                          ) : (
                            <div className="flex aspect-video items-center justify-center bg-alveus-green-50 text-xs text-alveus-green-300">
                              No Image
                            </div>
                          )}
```
`fill` tends to be rather problematic w/ next/image (as I imagine you discovered) due to it using `absolute` -- I think we'd be better off to just set `w-full` on the image + `aspect-video` directly. As an aside, we should also pass an explicit width to the image, as this'll determine at what size the optimization proxy renders it at (`w-full` will override this).

Reply:
(As an aside, not totally convinced on the hover effect here, I feel it implies that you're able to click on the image/card as a whole)

Reply:
> (As an aside, not totally convinced on the hover effect here, I feel it implies that you're able to click on the image/card as a whole)

Yeah, the hover effect is from before adding the buttons to copy to clipboard or run the command directly, clicking the card originally just ran the command.